docs/modules/ROOT/pages/object-storage/create.adoc

```diff
@@ -29,6 +29,17 @@ spec:
 <4> Secret where the connection details are provisioned. This secret shouldn't exist at this time.
 +
 IMPORTANT: Object Storage Bucket names must be unique across providers. Make sure to choose a name with prefixes and other guards, to prevent any issue during the creation of the bucket. No error message is returned if the bucket cannot be created.
++
+[NOTE]
+===============================
+A default Cloud Provider is defined per cluster but the user can choose between 2 providers available Exoscale or Cloudscale. For instance, to create an ObjectBucket on Cloudscale, add the following yaml snippet to the `spec` of an ObjectBucket:
+
+[source,yaml]
+----
+compositionRef:
+  name: cloudscale.objectbuckets.appcat.vshn.io
+----
+===============================
```

Comment:
I would create a new section and provide more details and use cases.
```suggestion
== Create a Bucket in another cloud provider

By default the bucket is created on the same cloud provider where the cluster is running on.
For instance, if you are on an Exoscale cluster, to create an ObjectBucket on cloudscale.ch add the following YAML snippet to the `spec` of an ObjectBucket:

[source,yaml]
----
spec:
  compositionRef:
    name: cloudscale.objectbuckets.appcat.vshn.io <1>
  parameters:
    region: rma <2>
----
<1> The reference name of the backing cloud provider
<2> Change the region identifier in accordance for the resp. cloud provider

[TIP]
====
This mechanism allows advanced deployment or failover scenarios.
For example, if your application is running on Exoscale, you can create a primary bucket on Exoscale and a secondary bucket on cloudscale.ch or vice versa.
Using a mirroring tool like https://rclone.org/[rclone] you can achieve bucket replication between different cloud providers.
====
```

Reply:
Great, thanks for suggestion.